Makefile

```diff
@@ -0,0 +1,5 @@
+lint:
+	flake8 place tests
+
+test:
+	py.test tests --cov=place --cov-report term-missing --cov-fail-under=100 --cov-branch
```

Comment:
This also seems cool, but I don't know if I understand.

Reply:
Is this pytest?

Reply:
Just run unit tests with py.test (instead of nosetest). Calculate coverage over on the `place` module (using branch coverage) and fail if test coverage isn't 100%.

Reply:
Sounds good.